frontend/src/app/list/components/list-share-dialog/list-share-dialog.component.ts

```
@@ -76,6 +77,7 @@ export class ListShareDialogComponent implements OnInit {
   displayedColumns: string[] = ['user', 'permission', 'actions'];
   permissionTypes = Object.values(Permission);
   readonly isLoading = signal(false);
+  isLoading = false;
```

Biome (2.4.6)

[error] Duplicate class member name "isLoading"

(lint/suspicious/noDuplicateClassMembers)